.travis.yml

```diff
@@ -7,12 +7,12 @@ php:
   - hhvm
 
 env:
-  - SYMFONY_VERSION=2.6.*
+  - SYMFONY_VERSION="~2.3||~3.0"
 
 matrix:
   include:
     - php: 5.3.3
-      env: SYMFONY_VERSION=2.3.* COMPOSER_FLAGS="--prefer-lowest"
+      env: SYMFONY_VERSION=2.3.* COMPOSER_FLAGS="--prefer-lowest --prefer-stable"
```

Comment:
Personally I don't get your argument inline with any real-life situation, but I'm reverting this, it's not the topic of this PR you're right!